src/gwt/src/org/rstudio/core/client/command/ShortcutManager.java

```diff
@@ -471,6 +475,28 @@ public void execute()
       
       keyBuffer_.add(keyCombination);
       
+      // we handle `Ctrl+L` for the sublime mode here
+      // because it is conflicted with `consoleClear`
+      if (editorMode_ == KeyboardShortcut.MODE_SUBLIME)
+         {
+            if (keyCombination.getKeyCode() == KeyCodes.KEY_L &&
+                keyCombination.getModifier() == KeyboardShortcut.CTRL)
```

Comment:
Make sure you set `rebindable=false` on this command in `Commands.cmd.xml` since this binding is hard-coded (i.e. no matter what users change the binding to we'll always expand the selection here)

Reply:
But it is only activated for the sublime mode, how can I only apply `rebindable` to sublime mode?